src/include/cell.i

```diff
@@ -201,17 +209,26 @@ __cell_pack_addr_validity(WT_SESSION_IMPL *session, uint8_t **pp, wt_timestamp_t
  */
 static inline size_t
 __wt_cell_pack_addr(WT_SESSION_IMPL *session, WT_CELL *cell, u_int cell_type, uint64_t recno,
-  wt_timestamp_t newest_durable_ts, wt_timestamp_t oldest_start_ts, uint64_t oldest_start_txn,
+  wt_timestamp_t stop_durable_ts, wt_timestamp_t oldest_start_ts, uint64_t oldest_start_txn,
   wt_timestamp_t newest_stop_ts, uint64_t newest_stop_txn, size_t size)
 {
+    wt_timestamp_t start_durable_ts;
     uint8_t *p;
+    bool prepare;
+
+    /*
+     * These values is set to default when packing, they are only set in future releases. We
```

Comment:
"These values is set" -> "These values are set"

Reply:
Fixed.